packages/storybook/package.json

```diff
@@ -58,6 +58,7 @@
   "scripts": {
     "storybook": "start-storybook -p 9009 -s public",
     "build-storybook": "NODE_OPTIONS='--max-old-space-size=4096' build-storybook -s public --quiet",
+    "build-storybook:watch": "NODE_OPTIONS='--max-old-space-size=4096' build-storybook -w -s public --quiet",
```

Comment:
The `README` says to use `npm run storybook`.. does this do something different?

Reply:
Yes, this is helpful when developing to have the storybook auto-rebuild when the code changes. The README is correct, but personally I prefer to work like this.